help_to_heat/frontdoor/interface.py

```diff
@@ -250,6 +252,11 @@ def create_referral(self, session_id):
             given_answers[loft_access_field] = loft_access_field_no_loft
             given_answers[loft_insulation_field] = loft_insulation_field_no_loft
 
+        # override "no insulation" to "less than threshold"
```

Comment:
Should ideally explain why we're doing this rather than what we're doing (can get the latter from the code). Context from the ticket should be enough.

Reply:
I've added some additional context :)